apps/experiments/models.py

```diff
@@ -53,19 +53,41 @@ def format_array_field(arr: list) -> str:
         return ", ".join([entry for entry in arr])
 
     @staticmethod
-    def format_trigger(trigger) -> str:
-        string = "If"
-        if trigger.trigger_type == "TimeoutTrigger":
-            seconds = seconds_to_human(trigger.delay)
-            string = f"{string} no response for {seconds}"
-        else:
-            string = f"{string} {trigger.get_type_display().lower()}"
-
-        trigger_action = trigger.action.get_action_type_display().lower()
-        return f"{string} then {trigger_action}"
+    def format_trigger(triggers) -> str:
```

Comment:
I'm surprised this doesn't need the same checks as `format_route` (`isinstance(triggers, VersionField)` and `isinstance(triggers, trigger_class)`